src/main/java/org/xbill/DNS/RRset.java

```diff
@@ -160,13 +160,13 @@
  * Returns an Iterator listing all (data) records.  This cycles through
  * the records, so each Iterator will start with a different record.
  */
-public synchronized Iterator
+public synchronized Iterator<Record>
 rrs() {
 	return iterator(true, true);
 }
 
 /** Returns an Iterator listing all signature records */
-public synchronized Iterator
+public synchronized Iterator<Record>
```

Comment:
These are all RRSIGRecords and can thus be returned like that. The casts are not source-compatible anyway if used like e.g. https://github.com/ibauersachs/dnssecjava/blob/master/src/main/java/org/jitsi/dnssec/validator/ValUtils.java#L404